### DISCUSSION TITLE
el-input设置prefixIcon属性不生效

### DISCUSSION THREAD

**Discussion by a1026546013** [element-plus/element-plus#8240] [Q&A]
已经引入过 icon图标

![image](https://user-images.githubusercontent.com/86405866/173308877-e90f070e-913f-4aa8-aae9-49af246ea504.png)
控制台出现警告
![image](https://user-images.githubusercontent.com/86405866/173309068-0c75240d-47a5-4d91-a949-0b4749a5bd05.png)
求大佬解答

**Comment by plainheart** [comment]
似乎没有复现该问题

**Comment by gizmo-ds** [comment]
提供下使用的包的版本, 代码也别用截图的, 也许会对复现你遇到的问题更有帮助.
